src/sections/Hero/Hero.tsx

```diff
@@ -95,7 +76,8 @@ export const Hero = () => {
   return (
     <FeaturedCase>
       <h2 style={{ fontWeight: 900 }}>
-        Empower your users with <TextUnderline>seamless form building</TextUnderline>
+        Empower your users with
+        <TextUnderline>seamless form building</TextUnderline>
```

Comment:
![Screenshot from 2023-07-28 12-15-42](https://github.com/vazco/forminer/assets/42475735/18bc8464-200b-4ef7-9aa1-829ad7a5b00e)


There's no space between `with` and `seamless` :fearful: